llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -590,6 +593,29 @@ bool SPIRVInstructionSelector::spvSelect(Register ResVReg,
   case TargetOpcode::G_USUBSAT:
     return selectExtInst(ResVReg, ResType, I, CL::u_sub_sat);
 
+  case TargetOpcode::G_SADDO:
+    report_fatal_error(
+        "Selection of a signed addition with overflow is not implemented",
```

Comment:
Instead of report_fatal_error, can you allow this to fallback to the generic expansion code?

Reply:
thank you, done